### PR TITLE
Avoid attributeError crash in xonfig wizard, V2.

### PR DESCRIPTION
**Pull request by bobhy**
*Real* fix for #3711.
Replaces PR #3712, which somehow got random other changes mixed into it.

--------------------------------------------

Looks like changes in #3704 didn't ripple outward to xonfig.py. It was using old field names.

Also https://github.com/xonsh/xonsh/blob/528d2f913ae40c89420fad02c665f3916aae44a0/xonsh/environ.py#L1867-L1877 would return unconverted `DefaultNotGiven` in `.doc_default` if called with environment variable not defined in `env._vars`.

### CODE REVIEW

File: xonsh/environ.py [hunk 1]
```diff
@@ -1865,12 +1865,10 @@ def get_default(self, key, default=None):
...
 
     def get_docs(self, key, default=None):
-        """Gets the documentation for the environment variable."""
+        """Gets the documentation for the environment variable. May be None"""
         vd = self._vars.get(key, None)
         if vd is None:
-            if default is None:
-                default = Var()
-            return default
+            return None
```
Comment (scopatz) [inline xonsh/environ.py:1871]: Shouldn't this return `default`?

**Comment by scopatz** [comment]
Thanks for putting this in! I think there is one small issue

File: xonsh/environ.py (hunk 1 above)
Comment (bobhy) [reply xonsh/environ.py]: `get_docs()` needs to ensure returned *obj*.doc_default is `str`, need to fix up default.
But the latest commit is a cleaner way to handle this, and lets me to remove the extra code where invoked in xonsh/xonfig.py (and where it should have been in the 2 other invocations I missed on the first go-round).

**Comment by scopatz** [comment]
Thanks @bobhy!

**Pull request merged**